docs/yellowpaper/sections/03-sphinx-packet-format/index.tex

```diff
@@ -34,36 +33,36 @@ \subsection{Construction}
 
 Let $r$ be the maximum number of nodes that a Sphinx mix message will traverse before being delivered to its destination.
 
-$G$ is a prime order cyclic group satisfying the decisional Diffie-Hellman assumption \cite{Boneh_1998}. We use the secp256k1 elliptic curve \cite{sec2}. The element $g$ is a generator of $G$ and $q$ is the (prime) order of $G$, with $q\approx2^{\kappa}$.
+$G$ is a prime order cyclic group satisfying the decisional Diffie-Hellman assumption \cite{Boneh_1998}. We use the secp256k1 elliptic curve \cite{sec2}. The element $g$ is a generator of $G$ and $q$ is the (prime) order of $G$, with $q\approx2^{2*\kappa}$.
 
 $G^*$ is the set of non-identity elements of G. $h_b$ is a pre-image resistant hash function used to compute blinding factors and modelled as a random oracle such that
 $h_b:G^*\times G^*\rightarrow\mathbb{Z}^*_q$, where $\mathbb{Z}^*_q$ is the field of non-identity elements of $\mathbb{Z}_q$ (field of integers). We use the BLAKE2s hash function \cite{blake2}.
 
 Each node $i$ has a private key $x_{i}\in \mathbb{Z}^*_q$ and a public key $y_{i}=g^{x_{i}}\in G^*$.
 $\alpha_i$ is the group elements which, when combined with the nodes’ public keys, allow a shared key to be computed for each via Diffie-Hellman (DH) key exchange. This ensures that each node in the user-chosen route can forward the packet to the next, and only the receiving mix node can decrypt it.
-$s_i$ are the DH shared secrets, $b_i$ are the blinding factors.
+$\varphi_i$ are the DH shared secrets, $b_i$ are the blinding factors.
 
-\paragraph{Key derivation}
+\subsection{Key derivation}
```

Comment:
these subsections should all be subsubsections as they're all under Constructions.